gradio/components/image.py

```diff
@@ -199,7 +202,7 @@ def postprocess(
             return None
         if isinstance(value, str) and value.lower().endswith(".svg"):
             return FileData(path=value, orig_name=Path(value).name)
-        saved = image_utils.save_image(value, self.GRADIO_CACHE)
+        saved = image_utils.save_image(value, self.GRADIO_CACHE, self.format or "png")
```

Comment:
Will this lead to loss of image quality when format is other than png?

Reply:
I suppose it depends on what the default JPG encoding settings are?